### PR TITLE
Routing_IPv6_base_functionality

### PR DESCRIPTION
**Pull request by olatunjikelani**
Signed-off-by: Kelani, Olatunji <olatunji.kelani@dell.com>

### CODE REVIEW

File: playbook/Routing_IPv6_base_functionality/roles/Routing_IPv6/tasks/Routing_IPv6_base_functionality.yml [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+    register: show_route
+    when: inventory_hostname == "Server1"
+  - debug: msg="{{show_route.stdout}}"
+    when: inventory_hostname == "Server1"
+  - name: TC3 - Enable IPv6 Forwarding - DUT
+    command: bash -c 'echo "1" > /proc/sys/net/ipv6/conf/all/forwarding'
+    register: ipv6_forwarding_server1
+    when: inventory_hostname == "Server1"
+  - name: TC3 - Configure Route - TR
+    command: bash -c "{{config_route_via_cps}} create --vrf_name default --dst '{{server1_ip}}/{{MASK}}' --nh_addr '{{DUT1_IPV6_ADDRESS_1}}'"
```
Comment (Tejaswi-Goel) [inline playbook/Routing_IPv6_base_functionality/roles/Routing_IPv6/tasks/Routing_IPv6_base_functionality.yml:399]: Cannot find where config_route_via_cps variable is defined?

File: playbook/site.yml [hunk 2]
```diff
@@ -56,5 +95,39 @@
...
+
+- hosts: 1host
+  gather_facts: false
+  become: true
+  pre_tasks:
+     - setup:
+  vars_files:
+    - "group_vars/{{ ansible_network_os }}_vars.yml"
+  roles:
+    - {role: Platform/roles/platform_watchdog, tags: ['platform_watchdog']}
```
Comment (Tejaswi-Goel) [inline playbook/site.yml]: Please add only Routing_IPv6_base_functionality entry, remove all other extra stuff.

File: playbook/site.yml [hunk 3]
```diff
@@ -56,5 +95,39 @@
   become: true
   pre_tasks:
      - setup:
+  vars_files:
+    - "group_vars/{{ ansible_network_os }}_vars.yml"
```
Comment (Tejaswi-Goel) [inline playbook/site.yml]: {{ ansible_network_os }} this variable is not used in opx-test.

**Comment by olatunjikelani** [comment]
Pull-request updated, HEAD is now a0e8a22892fbb5e3e88683ed41aefc4b1a46fb23

File: playbook/Routing_IPv6_base_functionality/roles/Routing_IPv6/tasks/Routing_IPv6_base_functionality.yml (hunk 1 above)
Comment (olatunjikelani) [reply playbook/Routing_IPv6_base_functionality/roles/Routing_IPv6/tasks/Routing_IPv6_base_functionality.yml:399]: it is defined in the group_var/all file.

Comment (Tejaswi-Goel) [reply playbook/Routing_IPv6_base_functionality/roles/Routing_IPv6/tasks/Routing_IPv6_base_functionality.yml:399]: thanks for adding that

File: playbook/site.yml [hunk 4]
```diff
@@ -47,6 +73,19 @@
...
+    - {role: L3/roles/data_vrf_basic, tags: ['data_vrf_basic']}
+
+
+- hosts: 2host
+  gather_facts: false
+  become: true
+  pre_tasks:
+     - setup:
+  vars_files:
+    - "group_vars/{{ ansible_network_os }}_vars.yml"
```
Comment (Tejaswi-Goel) [inline playbook/site.yml]: please undo these changes also, as {{ansible_network_os}} is not required

**Review by madhu222 — DISMISSED** [review]
Looks good to me.

File: playbook/site.yml [hunk 5]
```diff
@@ -47,14 +73,37 @@
   become: true
   pre_tasks:
      - setup:
+  vars_files:
+    - "group_vars/{{ ansible_network_os }}_vars.yml"
   roles:
-    - {role: L2/roles/l2_stp, tags: ['l2_stp']}
+    - {role: L3/roles/data_vrf_basic, tags: ['data_vrf_basic']}
```
Comment (Tejaswi-Goel) [inline playbook/site.yml]: Please remove data_vrf_basic entry.

**Review by Tejaswi-Goel — CHANGES_REQUESTED** [review]
